doc/api/process.md

```diff
@@ -1103,6 +1103,27 @@ and [Cluster][] documentation), the `process.connected` property will return
 Once `process.connected` is `false`, it is no longer possible to send messages
 over the IPC channel using `process.send()`.
 
+## `process.constrainedMemory()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* {number}
+
+Gets the amount of memory available to the process (in bytes) based on
+limits imposed by the OS. If there is no such constraint, or the constraint
+is unknown, `undefined` is returned.
+
+It is not unusual for this value to be less than or greater than `os.totalmem()`.
+This function currently only returns a non-zero value on Linux, based on cgroups
```

Comment:
This also needs to be updated since the return value is not zero anymore.

Reply:
I just remove this, the user can click the link to see details.